Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -122,6 +123,36 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
             this._fontVariationsGroup.rows = [emptyRow];
         }
+
+        // Custom Font Properties
+
+        // Display message when we have no custom fonts.
+        if (!this.nodeStyles.customFonts.length) {
+            this._customFontsRow.showEmptyMessage();
+            this._customFontsRow.dataGrid = null;
+            return;
+        }
+
+        // We can have no dataGrid if we navigate to a page
+        // with no custom fonts. This will trigger it to force
+        // rebuild the table.
+        if (!this._customFontsRow.dataGrid)
+            this._customFontsRow.dataGrid = this._dataGrid;
+
+        this._customFontsRow.dataGrid.removeChildren();
+
+        for (let customFont of this.nodeStyles.customFonts) {
+
+            // The backend can return empty font families, which we should ignore.
+            if (customFont.family === "")
```

Comment:
should we be filtering this in the backend?  im not sure i see a benefit in sending a font to the frontend if there's nothing to display for the name